Makefile

```diff
@@ -167,13 +167,16 @@ prepare-major: $(RELEASE_NOTES)
 # dependency and change the SUFFIX with the correct version so no need for
 # calling make on make is needed.
 # [1] https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html
-release: HANDLER_IMAGE_SUFFIX = :$(shell hack/version.sh)
-release: manifests push-handler $(description) $(GITHUB_RELEASE) version/version.go
+release: OPERATOR_IMAGE_SUFFIX =: $(shell hack/version.sh)
+release: HANDLER_IMAGE_SUFFIX =: $(shell hack/version.sh)
+release: manifests push-handler push-operator $(description) $(GITHUB_RELEASE) version/version.go
 	DESCRIPTION=$(description) \
 	GITHUB_RELEASE=$(GITHUB_RELEASE) \
 	TAG=$(shell hack/version.sh) \
 				   hack/release.sh \
-						$(shell find $(MANIFESTS_DIR) -type f)
+						$(shell find $(MANIFESTS_DIR) -type f) \
+						deploy/crds/nmstate.io_nmstates_crd.yaml \
```

Comment:
Why do we need these two CRDs? Aren't they in the MANIFESTS_DIR already?

Reply:
Nope, at manifests we have only the rendered stuff, CRDs are not passed to hack/render-manifests.go

Reply:
Gotcha, thanks